python/py-ezdxf/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup python        1.0
+
+name                    py-ezdxf
+version                 1.2.0
+revision                0
+
+checksums               rmd160  1eae28a34810f8e975755b0180cd6756d2154fc8 \
+                        sha256  a34748ac31d414387dab72ecf819b1422b44d43b094045a5235f2ad268882942 \
+                        size    2178069
+
+maintainers             {@flyingsamson tuxcad.de:flyingsamson} openmaintainer
+description             Python interface to DXF
+long_description        This Python package is designed to facilitate the creation and manipulation \
+                        of DXF documents, with compatibility across various DXF versions. It empowers \
+                        users to seamlessly load and edit DXF files while preserving all content, \
+                        except for comments.
+license                 MIT
+
+homepage                https://ezdxf.mozman.at
+
+python.versions         38 39 310 311 312
```

Comment:
is this intended to be used as a python module (i.e., imported in a Python shell or used by other packages) or as a stand-alone package that happens to be written in Python. If the former, than the use use as `py-ezdxf` with different Python versions is correct, otherwise it should probably just use the latest Python version and just be called `ezdxf`. If you can comment on this, then we can help out if it turns out that something would need to be changed.

Reply:
I'm currently in the process of packaging [flatcam](http://flatcam.org) in macports, which will be a standalone python program and the ezdxf package is one of its dependencies that was currently missing in macports.

With "comment on this" you mean here in this PR? Or do you mean I should add a comment to the port file at this point?